climpred/classes.py

```diff
@@ -84,6 +87,43 @@ def _display_metadata(self):
     return summary
 
 
+def _display_metadata_html(self):
+    header = f'<h4>climpred.{type(self).__name__}</h4>'
+    display_html(header, raw=True)
+    init_repr_str = dataset_repr(self._datasets['initialized'])
+    init_repr_str = init_repr_str.replace(
+        'xarray.Dataset', 'climpred Initialized : xarray.Dataset'
+    )
+    display_html(init_repr_str, raw=True)
+
+    if isinstance(self, HindcastEnsemble):
+        if any(self._datasets['observations']):
+            for key in self._datasets['observations']:
+                obs_repr_str = dataset_repr(self._datasets['observations'][key])
+                obs_repr_str = obs_repr_str.replace(
+                    'xarray.Dataset', f'climpred Verification {key} : xarray.Dataset'
+                )
+                display_html(obs_repr_str, raw=True)
+    elif isinstance(self, PerfectModelEnsemble):
+        if any(self._datasets['control']):
+            control_repr_str = dataset_repr(self._datasets['control'])
+            control_repr_str = control_repr_str.replace(
+                'xarray.Dataset', 'climpred Control : xarray.Dataset'
```

Comment:
Here, `Control Simulation`. I don't think we need the `xarray.Dataset` throughout or the `climpred` reference. Especially since the main header clarifies that this is a `climpred` object.